modules/openapi-generator/src/main/resources/php/ObjectSerializer.mustache

```diff
@@ -354,8 +354,8 @@ class ObjectSerializer
                     continue;
                 }
 
-                $propertyValue = $data->{$instance::attributeMap()[$property]};
-                if (isset($propertyValue)) {
+                if (isset($data->{$instance::attributeMap()[$property]})) {
+                    $propertyValue = $data->{$instance::attributeMap()[$property]};
```

Comment:
The previous statement basically made no sense, it's just a null check in disguise.